hack/generate-mocks.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+echo Install GoMock
+go install github.com/golang/mock/mockgen@v1.6.0
+
+echo Generate GoMock helper files
+mockgen --build_flags=--mod=mod -destination mock/mock_capabilities.go -package mock_capabilities github.com/kubewarden/policy-sdk-go/capabilities WapcClient
```

Comment:
As far as I can see, you are using the reflect mode of mockgen. I have an question. If I change the WapcClient locally, it will consider the local version or the stable one published in the repo? If it's the repo one, we should change the call to use the local one. Can we say to mockgen, using the reflection mode, to check in the local copy? If not, we may need to use the source code mode.

Reply:
Good question. Yes, I'm using reflect mode. I just tried it and it is using the local version. :)